JupyterDocs/ariaMisclosure/ariaMisclosure_tutorial.ipynb

```diff
@@ -1,10 +1,5 @@
 {
```

Comment:
<p>same. think you did not run cells, but would be good to check if this still works.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aria-tools/ARIA-tools-docs/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>